### PR TITLE
Rerun MathJax for staff grade view

### PR DESCRIPTION
**Pull request by geevi**
**TL;DR -** [ A short summary of what this PR does and why ]
In the staff grading view, math formatted writeup was not working. MathJax.Hub.Typeset() needs to be called for typesetting, dynamically loaded part. 

**What changed?**
Just added a script tag to the template which calls MathJax.Hub.Typeset()

- [ More in depth breakdown of changes ]
- [ Peripheral things that got changed ]
- [ etc... ]

**Developer Checklist**
- [ ] Reviewed the [release process](./release_process.md)
- [ ] Translations up to date
- [ ] JS minified, SASS compiled
- [ ] Test suites passing on Jenkins
- [ ] Bumped version number in [setup.py](../setup.py) and [package.json](../package.json)

JIRA: [JIRA-XXXX](https://openedx.atlassian.net/browse/JIRA-XXXX)

FIY: @edx/masters-devs-gta

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @geevi! I've created [OSPR-4949](https://openedx.atlassian.net/browse/OSPR-4949) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

Please let us know once your PR is ready for our review and all tests are green.

:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://open.edx.org/wp-content/uploads/2019/01/individual-contributor-agreement.pdf)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.

**Comment by natabene** [comment]
@geevi Thank you for the contribution! When you have a chance, could you please send in a signed Contributor Agreement mentioned in the earlier comment? You only need to do this once and we can then review all your future code contributions. Thanks

**Comment by geevi** [comment]
@natabene I have send the signed CLA to the email mentioned in it. Let me know if anything else needs to be done. 

Also I think this issue of math rendering exists at many place in the ORA. Whenever resposes are loaded dynamically MathJax.Hub.Typeset() needs to be called.

**Comment by natabene** [comment]
@geevi Thank you for signing and sending. Once our Legal team processes it, we will inform you and you will be all set for us to start a review.

**Comment by natabene** [comment]
@geevi Have you had a chance to finalize signing with our Legal team?

**Comment by natabene** [comment]
@pomegranited Yes, I think that should be fine for you to review this. CC @nedbat @feanil

**Comment by pomegranited** [comment]
@geevi I'm unable to get ORA2 to display formatted MathJax at all.. could you attach a course export tar.gz which demonstrates this feature?

**Comment by natabene** [comment]
@geevi Just checking, will you have a chance to provide the requested information?

**Comment by natabene** [comment]
@geevi We have not heard from you in a while, so I am closing this PR for now. Please feel free to re-open if you decide to continue working on this.

**Comment by edx-webhook** [comment]
@geevi Even though your pull request wasn’t merged, please take a moment to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=edx/edx-ora2&entry.867055334=https://github.com/edx/edx-ora2/pull/1488&entry.1484655318=https://github.com/geevi&entry.752974735=2020-09-05+14:22&entry.1917517419=2021-02-10+18:34&entry.2133058324=No) so we can improve your experience in the future.
